verilog/CST/port_test.cc

```diff
@@ -252,5 +256,43 @@ TEST(GetIdentifierFromTaskFunctionPortItemTest, ExpectSomeTaskFunctionPorts) {
   }
 }
 
+TEST(GetAllPortReferences, GetPortReferenceIdentifier) {
+  constexpr int kTag = 1;  // value doesn't matter
+  const SyntaxTreeSearchTestCase kTestCases[] = {
+      {""},
+      {"module m(); endmodule: m"},
+      {"module m(",
+       {kTag, "a"},
+       ", ",
+       {kTag, "b"},
+       ");\n input a, b; endmodule: m"},
+      {"module m(input a,", {kTag, "b"}, "); endmodule: m"},
+      {"module m(input a,", {kTag, "b"}, "[0:1]); endmodule: m"},
+      {"module m(input wire a,", {kTag, "b"}, "[0:1]); endmodule: m"},
```

Comment:
For future work (new issue, if there isn't one already):

Facts on this example should include that `b` has the same direction and type as `a`: `input` and `wire`.